README.md

```diff
@@ -0,0 +1,37 @@
+[![GoDoc](https://godoc.org/github.com/m-lab/bmctool?status.svg)](https://godoc.org/github.com/m-lab/bmctool) [![Build Status](https://travis-ci.org/m-lab/bmctool.svg?branch=master)](https://travis-ci.org/m-lab/bmctool) [![Coverage Status](https://coveralls.io/repos/github/m-lab/bmctool/badge.svg?branch=master)](https://coveralls.io/github/m-lab/bmctool?branch=master) [![Go Report Card](https://goreportcard.com/badge/github.com/m-lab/bmctool)](https://goreportcard.com/report/github.com/m-lab/bmctool)
+
+# BMCTool
+BMCTool is a command line tool to manage BMC credentials on the M-Lab infrastructure.
```

Comment:
Depending on the target audience, it may be worth expanding the BMC acronym or adding a link